backend/capellacollab/projects/crud.py

```diff
@@ -42,6 +43,11 @@ def create_project(
     db: Session, name: str, description: str | None = None
 ) -> DatabaseProject:
     slug = slugify(name)
+    if (
+        db.query(DatabaseProject).filter(DatabaseProject.slug == slug).first()
+        is not None
+    ):
+        raise HTTPException(409, "Slug already used.")
```

Comment:
With the comment above, the error message is not user friendly. `slug` is too technical, therefore I would recommend the same message as in the frontend as displayed message:  `A project with a name similar to <slug> already exists.`.

However, we should still keep the current message as technical message - to give developers a more detailed feedback.